add_translation.rb

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
```

Comment:
Let's keep this one out, I'll add it to another repository which is more suitable for this type of scripts

Reply:
Fine, I'll rebase and delete the commit that adds it. I've sent you the code via WhatsApp. 🙈